dbdialog/src/org/pentaho/ui/database/DatabaseConnectionDialog.java

```diff
@@ -43,10 +43,9 @@ public void registerClass( String key, String className ) {
     extendedClasses.put( key, className );
   }
 
-  public XulDomContainer getSwtInstance( Shell shell ) throws XulException {
+  public XulDomContainer getSwtInstance( SwtXulLoader loader, Shell shell ) throws XulException {
```

Comment:
In the kettle - yes. But there is DatabaseConnectionDialog in the pentaho-commons-database also. It's strange. It could be issue.